zun/container/k8s/driver.py

```diff
@@ -591,7 +650,7 @@ def get_archive(self, context, container, path):
 
     @utils.check_container_id
     def put_archive(self, context, container, path, data):
-        """Copy resource to a container."""
+        """Copy resource to a container. This utility requires the running container to have sh and tar."""
```

Comment:
Comment is good, and fortunately we havent introduced a new dependency, since the older implementation also used tar.